### ISSUE TITLE
High loss while training on Nuscenes

### ISSUE THREAD

**Issue by kaxapatel** [edwardzhou130/Panoptic-PolarNet#21]
Hi. Really good work. I am trying to train the model but I can not debug why the losses are very high and PQ is too poor. Hope you may have some idea.
![Screenshot from 2022-11-02 16-06-46](https://user-images.githubusercontent.com/95223869/199526523-188174a5-0140-4bd1-be41-0a759c0d9907.png)

**Comment by edwardzhou130** [comment]
I can't tell what the problem could be from the screenshot. Can you provide more information? For example:
1. Did you change any configs or code when training your model? 
2. Are you training on the SemanticKITTI dataset?
3. Is the loss decreasing in your training?

**Comment by kaxapatel** [comment]
1. No I have not changed anything just build a code from Polarnet for nuscenes data loader.
2. No this is on Nuscenes mini
3. no it is not decreasing. 
4. I am trying if without augumentation becuase neither I have build the instances for nuscenes nor made a .yaml file for labels.
5. I have changed this in dataloader for panoptic is that the correct way to read panoptic data?
![Screenshot from 2022-11-03 12-43-47](https://user-images.githubusercontent.com/95223869/199712322-3e439b7b-3feb-4d10-9259-0ab7fcdf98e9.png)

**Comment by edwardzhou130** [comment]
Sorry for the late reply. You don't need to add augmentation to train the model. So it is not the issue here. My guess is the annotation is not converted to the correct semantic label and instance id.  Can you check if the ``annotated_data`` you loaded contains the semantic label between 0-16 and a unique instance id for each instance? You also need to edit the .yaml file since the class is not the same as SemanticKITTI. https://github.com/edwardzhou130/Panoptic-PolarNet/blob/3a72f2380a4e505e191b69da596f521a9d9f1a71/semantic-kitti.yaml#L109-L206

**Comment by kaxapatel** [comment]
Ok. Thank You for your attention. I have one more query. I hase started working with custom dataset for my own project. I have velodyne, its label annotation and which if for landmark detection. so what changes I must do for training using your code. how can I get content, learning_map(if I have few distinct classes) and its inverse?
